modules/nsq/src/it/scala/com/snowplowanalytics/snowplow/enrich/nsq/test/Containers.scala

```diff
@@ -218,21 +218,24 @@ object Containers {
   ): Resource[F, JGenericContainer[_]] =
     Resource.make(
       Sync[F].delay {
-        val config = Config.base64EnrichConfig(
-          lookupHost = topology.lookup1.networkAlias,
-          lookupPort = topology.lookup1.httpPort,
-          nsqdHost = topology.nsqd1.networkAlias,
-          nsqdPort = topology.nsqd1.tcpPort,
-          inputTopic = topology.sourceTopic,
-          goodOutputTopic = topology.goodDestTopic,
-          badOutputTopic = topology.badDestTopic
-        )
         val container = GenericContainer(
           dockerImage = s"snowplow/snowplow-enrich-nsq:${BuildInfo.version}-distroless",
           env = Map(
             "JAVA_OPTS" -> "-Dorg.slf4j.simpleLogger.defaultLogLevel=info",
```

Comment:
I know that this is what we use for Kinesis tests, but I think that now that we use the `-distroless` images for the tests this is not correct any more and we have to use `JDK_JAVA_OPTIONS` instead. Could you check and update Kinesis as well if that's the case please?